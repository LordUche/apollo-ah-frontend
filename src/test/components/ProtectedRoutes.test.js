import React from 'react';
import setup from '../setup';
import ProtectedRoutes from '../../routes/ProtectedRoutes';

const container = <ProtectedRoutes activePage="PROFILE" />;

describe('<ProtectedRoutes>', () => {
  it('should render without crashing', () => {
    const wrapper = setup(container);
    expect(wrapper).toMatchSnapshot();
  });
});
