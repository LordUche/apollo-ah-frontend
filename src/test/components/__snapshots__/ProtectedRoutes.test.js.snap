// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProtectedRoutes> should render without crashing 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <BrowserRouter>
    <Router
      history={
        Object {
          "action": "REPLACE",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "key": "qb4xqv",
            "pathname": "/login",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <Connect(ProtectedContainer)
        activePage="PROFILE"
      >
        <ProtectedContainer
          activePage="PROFILE"
          dispatch={[Function]}
          isLoggedIn={false}
        >
          <Redirect
            push={false}
            to="/login"
          />
        </ProtectedContainer>
      </Connect(ProtectedContainer)>
    </Router>
  </BrowserRouter>
</Provider>
`;
